src/__tests__/SearchResults.test.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import SearchResults from "../components/SearchResults";
+
+describe("Search Results", () => {
+  it("renders correctly", () => {
+    const { asFragment } = render(<SearchResults results={[]} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
```

Comment:
This test is not working, partly because you are not passing in all the required props to the component, and also as you are trying to create 2 snapshots in the same file.
```
  const validProps = {
    hasSearched: jest.fn(),
    results: [],
  };

  it("renders correctly", () => {
    const { asFragment } = render(
      <SearchResults
        results={validProps.results}
        hasSearched={validProps.hasSearched}
      />
    );

    expect(asFragment()).toMatchSnapshot();
  });
```
This will create the snapshot of the file as it looks when the component is first mounted